examples/plugins/operator/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#------------------------------------------------------------------------------#
+# Distributed under the OSI-approved Apache License, Version 2.0.  See
+# accompanying file Copyright.txt for details.
+#------------------------------------------------------------------------------#
+
+list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_CURRENT_SOURCE_DIR})
+find_package(Sodium 1.8 REQUIRED)
```

Comment:
Is this compulsory or optional? If it's compulsory then we probably want to include libsodium in thirdparty, because we want users to be able to compile ADIOS2 out of box without having to manually install dependent libraries, at least for the basic BP engines.